slam-libraries/viam-cartographer/src/main_test.cc

```diff
@@ -0,0 +1,9 @@
+#define BOOST_TEST_MODULE cartographer tests
+#include <boost/test/included/unit_test.hpp>
+
+#include "glog/logging.h"
+
+struct GlobalFixture {
+    void setup() { google::InitGoogleLogging("main_test"); }
```

Comment:
Added this to suppress printouts when running tests, to make test results easier to read. Can always be commented out if printouts are desired.